semana3/blackjack/nossoJogo.js

```diff
@@ -0,0 +1,48 @@
+/**
+ * EXEMPLO DE UTILIZAÇÃO DA 'comprarCarta'
+ * 
+ * 
+    const carta = comprarCarta(); // Sorteia uma carta. Por exemplo, o rei de ouros
+    
+    console.log(carta.texto) // imprime o texto da carta. Exemplo: "K♦️" (indica "K" de ouros)
+    console.log(carta.valor) // imprime o valor da carta (um número). Exemplo: 10 (dado que "K" vale 10)
+ * 
+ * 
+ * 
+ */
+console.log("Bem vido ao jogo Blackjack!\n ")
+let pergunta = window.confirm("Quer iniciar uma nova rodada?")
+
+
+let cartas = []
+let nipes = []
+
+   if(pergunta){
+
+      for(i = 0; i < 4; i++){
+         const carta = comprarCarta()
+         cartas.push(carta.valor)
+         nipes.push(carta.texto)
+      }
```

Comment:
Bom uso do for. Mas talvez fosse melhor separar as cartas do usuario e as cartas do computador.